tests/models/dsheetpiling/test_dsheetpiling_acceptance.py

```diff
@@ -92,12 +89,14 @@
     Soil,
     SoilType,
 )
+from pydantic.color import Color
+from teamcity import is_running_under_teamcity
 from tests.utils import TestUtils, only_teamcity
 
 
 class TestDsheetPilingAcceptance:
+    # @only_teamcity
```

Comment:
Maybe we also delete the comments

Reply:
Not sure why you get to see this comment because in my latest revision it's actually not commented out.